CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
-## 3.2.1-dev -
+## 3.2.1 - 2023-06-01
+
+### Fixed
+- Fix bug where `env:clone-content` command was not working Drupal sites (#2465)
+- Fix bug where `env:clone-content` command was not working WordPress sites (BUGS-6307)
```

Comment:
is there a PR for this item? If not, I'd prefer referencing the same PR and updating it to have a reference to the bugs ticket than putting the bugs ticket here as it's not really meaningful for users

Reply:
There's not a PR. It was an internal bug. remove?

Reply:
So how did it got fixed? 🤔 

Also, for the L7, I think it would be better to point to the PR than the issue: https://github.com/pantheon-systems/terminus/pull/2469

Reply:
I guess this is the PR for L8: https://github.com/pantheon-systems/terminus/pull/2471